firmware/Core/Src/Logs.c

```diff
@@ -0,0 +1,36 @@
+#include "Logs.h"
+#include "esp8266.h"
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+void Logs_Init(void)
+{
+}
+
+int Logs_LogData(const LogData_t * LogData)
+{
+    if(Logs_WifiData(* LogData)):
```

Comment:
these functions were exposed in the interface. But you also call them from the inside of the module. I'd recommend that you call Logs_WifiData, Logs_FlashData, and Logs_LCDData from your state in Controller.c Not a big deal, but will make things slightly more explicit and make the debug effort easier. Either that or don't expose those and only expose Logs_LogData.

As a rule, only expose functions meant to be called from the higher level in the Interface. Any helper functions part of your module should be declared static inside the .c file.

Reply:
oh ok makes more sense thx